### PR TITLE
update devops observability

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: zhuxiaoyang <sunzhu@yunify.com>

### CODE REVIEW

File: developer-guide/concepts-and-designs/sig-devops/devops_observability.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Expose Pipiline metrics to Prometheus
+
+​	In order to have a more intuitive statistically view about pipelines. we should collect some metrics about pipelines, and then expose an endpoint with metrics where a Prometheus Server can scrape.
```
Comment (FeynmanZhou) [inline developer-guide/concepts-and-designs/sig-devops/devops_observability.md]: `pipelines. we should` => `pipelines, we may need to`

File: developer-guide/concepts-and-designs/sig-devops/devops_observability.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+​    Now we recommend using  [Jenkins Prometheus Plugin](https://github.com/jenkinsci/prometheus-plugin) to expose metrics about pipelines. Currently only metrics from the [Metrics-plugin](https://github.com/jenkinsci/metrics-plugin) and summary of build duration of jobs and pipeline stages.
```
Comment (FeynmanZhou) [inline developer-guide/concepts-and-designs/sig-devops/devops_observability.md]: Currently, only...

Comment (FeynmanZhou) [reply developer-guide/concepts-and-designs/sig-devops/devops_observability.md]: `Currently only metrics from the [Metrics-plugin](https://github.com/jenkinsci/metrics-plugin) and summary of build duration of jobs and pipeline stages. `, this sentence is very strange.

File: developer-guide/concepts-and-designs/sig-devops/devops_observability.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+| `jenkins_builds_success_build_count`                         | `counter` | Successful build count                                |      |
+| `jenkins_builds_failed_build_count`                          | `counter` | Failed build count                                    |      |
+| `jenkins_builds_last_build_result`                           | `gauge`   | Build status of a job as a boolean                    |      |
+| `jenkins_builds_last_build_duration_milliseconds`            | `gauge`   | Build times in milliseconds of last build             | ms   |
+| `jenkins_builds_last_build_start_time_milliseconds`          | `gauge`   | Last build start timestamp in milliseconds            | ms   |
+| `kubesphere_jenkins_builds_stage_duration_milliseconds_summary` | `summary` | Summary of Jenkins build times by Job and Stage       | ms   |
+| `jenkins_builds_last_build_result_ordinal`                   | ``        |                                                       |      |
+
+more info please to see **https://github.com/jenkinsci/metrics-plugin#metrics-plugin-for-jenkins**
```
Comment (FeynmanZhou) [inline developer-guide/concepts-and-designs/sig-devops/devops_observability.md]: Do users need to integrate this plugin manually?

Comment (soulseen) [reply developer-guide/concepts-and-designs/sig-devops/devops_observability.md]: no, we will install it default

**Comment by soulseen** [comment]
fixed

**Comment by rayzhou2017** [comment]
@runzexia Please focus on reviewing the content. Thanks

File: developer-guide/concepts-and-designs/sig-devops/devops_observability.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+## Expose Pipiline metrics to Prometheus
+
+​	In order to have a more intuitive statistically view about pipelines. we may need to collect some metrics about pipelines, and then expose an endpoint with metrics where a Prometheus Server can scrape.
```
Comment (runzexia) [inline developer-guide/concepts-and-designs/sig-devops/devops_observability.md:3]: we need？

Comment (soulseen) [reply developer-guide/concepts-and-designs/sig-devops/devops_observability.md:3]: yes, and this work is what the Jenkins Prometheus Plugin does.

**Comment by soulseen** [comment]
updated

**Comment by FeynmanZhou** [comment]
@zryfish Could you pls set a Prow bot for this repo? We can use command to do some actions.

**Pull request merged**